amxmodx/CLang.h

```diff
@@ -118,7 +113,7 @@ class CLangMngr
 	
 	LangVec m_Languages;
 
-	CVector<md5Pair *> FileList;
+	StringHashMap<time_t> FileList;
 	CVector<String *> KeyList;
 	THash<String, keytbl_val> KeyTable;
```

Comment:
I would do that in another PR. The following patch is just to fix the shit code on this specific chunk of code.
I will make a PR to clean the whole class.